cmake/autocoder/autocoder.cmake

```diff
@@ -134,22 +134,24 @@ function(run_ac BUILD_TARGET_NAME AUTOCODER_CMAKE SOURCES GENERATED_FILE_LIST HA
         else()
             __ac_process_sources("${BUILD_TARGET_NAME}" "${AC_INPUT_SOURCES}")
         endif()
-        # Read autocoder outputs from properties using the centralized variable lists
-        set(ALL_AUTOCODER_VARIABLES ${FPRIME_AUTOCODER_REQUIRED} ${FPRIME_AUTOCODER_OPTIONAL} AUTOCODER_GENERATED)
-        
-        # Process each autocoder variable and append to target properties with the same name
-        foreach(VARIABLE_NAME IN LISTS ALL_AUTOCODER_VARIABLES)
-            get_property(VARIABLE_VALUES TARGET "${BUILD_TARGET_NAME}" PROPERTY "${AUTOCODER_NAME}_${VARIABLE_NAME}")
-            if (VARIABLE_VALUES)
-                append_list_property("${VARIABLE_VALUES}" TARGET "${BUILD_TARGET_NAME}" PROPERTY "${HASH}_${VARIABLE_NAME}")
-            endif()
-        endforeach()
-        _describe_autocoder_run("${AUTOCODER_NAME}")
     else()
         # Assert runs are identical for the same autocoder
         fprime_cmake_ASSERT("Hash mismatch for autocoder ${AUTOCODER_NAME}: stored '${STORED_HASH}' vs calculated '${SRCS_HASH}'" 
                             "${STORED_HASH}" STREQUAL "${SRCS_HASH}")
     endif()
+
```

Comment:
This fix is correct.  Pulling the "read" of the properties out of the `if (not already run)` block ensures they are read in both cases.

The is a minor change in behavior in which the description of the run is also emitted even when run before, however; this is also likely desirable.